recipes/ibex/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ibex" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 989db0004432c50e237c34d99912d10c00552e2bb57e162de11799378d9bd40e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.5
```

Comment:
Is this package really only for `python >=3.5`? In the `setup.py` it seems to indicate python2 support:

https://github.com/atavory/ibex/blob/master/setup.py#L129

Reply:
I believe that supporting py2 would require the use of selectors which would preclude the use of `noarch: python`. Since no one is clamouring for a py2 conda package I applied the YAGNI principle.

I'm happy for a maintainer to step up and fix the recipe to also support py2 and if a user actually pops up requesting it I can look into it myself then. In the meantime I think it's fine putting less strain on the CI system until demand for py2 support actually materialises.

Reply:
I agree with you. 

If an user request a package for python 2 we will have to create a package for that. :)

Thanks to put this in conda-forge! I'm going to merge this.